pkg/controller/master/storagenetwork/storage_network.go

```diff
@@ -760,11 +758,9 @@ func (h *Handler) checkPodStatusAndStop() error {
 		return err
 	}
 
-	if err := h.checkVMImportControllerStatusAndStop(); err != nil {
-		return err
-	}
+	err := h.checkVMImportControllerStatusAndStop()
 
-	return nil
+	return err
```

Comment:
prefer to `return h.checkVMImportControllerStatusAndStop()`

Reply:
I will consider to keep the original format (before codefactor commit)
but I will change it.